wandb/apis/public/api.py

```diff
@@ -1048,3 +1048,24 @@ def list_jobs(self, entity, project):
             return [x["node"]["artifacts"] for x in artifacts]
         except requests.exceptions.HTTPError:
             return False
+
+    @normalize_exceptions
+    def exists(self, name, type=None):
+        _, _, artifact_name = self._parse_artifact_path(name)
+        if ":" in artifact_name:
+            try:
+                self.artifact(name, type)
+                return True
+            except wandb.errors.CommError:
+                return False
+        else:
+            if type is None:
+                raise ValueError(
+                    "You must specify type= to check if there are any artifacts in this collection"
+                )
+            try:
+                artifacts = self.artifacts(type, name, 1)
+                len(artifacts)
+                return True
```

Comment:
```suggestion
                return len(artifacts) > 0
```
I think this might be necessary in the case where a collection exists but all members have been deleted (I'm not positive, but I think you might get an empty response back instead of an error).

Reply:
yeah `len(artifacts)` will return 0 in that case, not error out. didn't realize we'd still want to return false here, will change this!